test/dlc_tests/ec2/pytorch/training/common_cases.py

```diff
@@ -20,24 +20,30 @@
 
 # Test functions
 PT_STANDALONE_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "pytorch_tests", "testPyTorchStandalone")
```

Comment:
Reorder functions, added TE, GDRCopy, and `PT_COMMON_GLOO_MPI_CMD` and `PT_COMMON_NCCL_CMD` since those two are using the same command but different input for `gloo`/`mpi`/`nccl` + with/without inductor